app/controllers/katello/api/v2/organizations_controller.rb

```diff
@@ -73,6 +73,7 @@ def create
       @organization = Organization.new(resource_params)
       sync_task(::Actions::Katello::Organization::Create, @organization)
       @organization.reload
+      @taxonomy = @organization
```

Comment:
This is definitely the type of code that warrants a comment since it could easily be deleted and we wouldn't be aware of anything breaking on the Foreman side - mind adding one?